addons/account_edi/models/account_move.py

```diff
@@ -364,15 +364,13 @@ def action_process_edi_web_services(self, with_commit=True):
         docs = self.edi_document_ids.filtered(lambda d: d.state in ('to_send', 'to_cancel') and d.blocking_level != 'error')
         docs._process_documents_web_services(with_commit=with_commit)
 
-    def _retry_edi_documents_error_hook(self):
-        ''' Hook called when edi_documents are retried. For example, when it's needed to clean a field.
-        TO OVERRIDE
```

Comment:
This method is used in l10n_co_edi soooo don't think we should remove it ? or at least update the code there

Reply:
its handled in the following PR already https://github.com/odoo/enterprise/pull/88173 (make sure to r+ it with this one)

Reply:
Ahh didn't saw the enterprise pr ahah, oupsss